src/server/routes/api/classrooms/index.js

```diff
@@ -293,23 +338,44 @@ router.post("/:classroomId/answer", async (req, res) => {
     });
   }
 
-  const question = classroom.questions.filter(q => q._id.toString() === _id)[0];
+  const question = classroom.questions.filter(q => q._id.toString() === questionId)[0];
```

Comment:
I know I put this in originally, but while you're making changes `classroom.questions.id(questionId)` is a much more elegant way to do this.